libs/core/src/lib/shellbar/user-actions-menu/user-actions-menu/user-actions-menu.component.ts

```diff
@@ -0,0 +1,105 @@
+import { ChangeDetectionStrategy, Component, ContentChildren, EventEmitter, Input, Output, QueryList, ViewChild, ViewEncapsulation } from '@angular/core';
+import { Placement, PopperOptions } from 'popper.js';
+
+import { PopoverComponent, PopoverFillMode } from '../../../../lib/popover/public_api';
+import { ShellbarUser } from '../../model/shellbar-user';
+import { UserActionsMenuItemComponent } from '../user-actions-menu-item/user-actions-menu-item.component';
+
+@Component({
+  selector: 'fd-user-actions-menu',
+  templateUrl: './user-actions-menu.component.html',
+  styleUrls: ['./user-actions-menu.component.scss'],
+  encapsulation: ViewEncapsulation.None,
+  changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class UserActionsMenuComponent {
```

Comment:
If you want to get rid of all of these inputs, you can extend base popover class from `libs/core/src/lib/popover/base/base-popover.class.ts`.